strawberry/tools/create_type.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import field as dataclasses_field, make_dataclass
+from typing import List, Type, cast
+
+import strawberry
+from strawberry.field import StrawberryField
+
+
+def create_type(name: str, fields: List[StrawberryField]) -> Type:
```

Comment:
let's add a docstring 😊

do you think we should add some safety checks here? I'm thinking of checking at runtime if fields are actual fields and also preventing mixing fields with mutations, what do you think?

Reply:
Runtime checks sound like a good idea 👍

Reply:
Also maybe a check to make sure `fields` is not empty? (at least when creating a Query 🤔 )

Reply:
@marcoacierno we can't tell if you're creating a Query (since it's the same as any other type) but I think it makes sense to always raise an exception if the function is passed an empty list of fields.

Reply:
@patrick91 I like the idea of preventing fields and mutations but AFAIK currently we can't tell if a field is a mutation or now: https://github.com/strawberry-graphql/strawberry/blob/f31e6c17238ba4372129c4f291ddf336b4e6b901/strawberry/mutation.py#L8